docs/http-body.md

```diff
@@ -0,0 +1,53 @@
+# Body
+
+## Standard properties
+
+Error responses from publiq APIs always include at least the 4 following properties defined by the [RFC7807](https://datatracker.ietf.org/doc/html/rfc7807) standard:
+
+*   `type` - A URL used to identify the problem. You can use this to differentiate between errors inside your application. Make sure to always compare the full URL and not just the last part, since some APIs might share similar yet different error types.
+*   `title` - A short, human-readable summary of the problem type.
+*   `detail` - A human-readable summary about this specific instance of the problem.
```

Comment:
detail include ik niet by default (volgens de spec is die ook optioneel)

Reply:
Ah oke, ik had begrepen dat je die altijd ook include. Zal die als optioneel zetten dan